### PR TITLE
Add README.md and basic docs

### PR DESCRIPTION
**Pull request by rooftopcellist**
This PR adds a readme with basic docs on installation and usage of the awx-resource-operator. 

Currently, I have the operator version on my own public quay.  I will get a domain for this shortly and update it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+### Prerequisites
+
+* Install the kubernetes-based cluster of your choice:
+  * [Openshift](https://docs.openshift.com/container-platform/4.11/installing/index.html)
+  * [K8s](https://kubernetes.io/docs/setup/)
+  * [CodeReady containers](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5)
+  * [minikube](https://minikube.sigs.k8s.io/docs/start/)
+* Deploy AWX using the [awx-operator](https://github.com/ansible/awx-operator#basic-install)
+* Create an OAuth2 token for your user in the AWX UI
```
Comment (rlopez133) [inline README.md:40]: Add steps for creating OAuth2 token

** Via the AWX UI:
  * Login to AWX with admin permissions
  * Navigate to Users
  * Select the username you wish to create a token for
  * Click on tokens, then the green plus icon
  * Application can be left empty, input a description and select a scope <r/w>

File: README.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+```
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: awx-access
+  type: Opaque
+stringData:
+  token: X4yryQk1FzTCmglqPuPblahblahuQXT
```
Comment (rlopez133) [inline README.md]: change this to:

`token: <generated-token>`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+```
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: awx-access
+  type: Opaque
+stringData:
+  token: X4yryQk1FzTCmglqPuPblahblahuQXT 
+  host: https://awx.apps.dev.ocp.me.com/
```
Comment (rlopez133) [inline README.md]: change this to:

`host: https://my-awx-host.example.com/`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Create a job template on AWX by creating a JobTemplate resource.
+
+```
+---
+apiVersion: tower.ansible.com/v1alpha1
+kind: JobTemplate
+metadata:
+  name: jobtemplate-4
+spec:
```
Comment (rlopez133) [inline README.md:168]: Job templates have so many more values e.g. credentials, forks. Options to add those exist or no?

**Review by rlopez133 — CHANGES_REQUESTED** [review]
Chris this looks really great! Added some comments and feedback.

File: README.md (hunk 4 above)
Comment (rooftopcellist) [reply README.md:168]: These are the only options that will be passed through to the runner process afaik.  Other options can be supported in the future, but would need to be added to the AnsibleJob spec and passed through.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+## Usage
+
+Now that the awx-resource-operator is installed in your cluster, you can try creating a k8s resource to manage your AWX instance.
+
+First, you need create a k8s secret with connection information for your AWX instance though. Below is example yaml that you can save to a file, edit, and apply to your cluster.
+
+### Create an AWX Access Token
+
+Create an OAuth2 token for your user in the AWX UI.
```
Comment (rooftopcellist) [inline README.md:74]: @rlopez133 I added the steps to create a personal access token in it's own section here.

Comment (rlopez133) [reply README.md:74]: Looks good

File: README.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+  inventory: Demo Inventory                # Inventory prompt on launch needs to be enabled
+  runner_image: quay.io/ansible/awx-resource-runner
+  runner_version: latest
+  job_ttl: 100
+
+  extra_vars:                              # Extra variables prompt on launch needs to be enabled
+     test_var: test
+```
+
+Note that prompt on launch needs to be enabled for inventories and extra vars if you are specifying those.
```
Comment (rlopez133) [inline README.md]: (Unsure if there is a way to do this programmatically regarding prompt launch)

But I'd add:

In order to enable "PROMPT ON LAUNCH", within the AWX UI:
* Select TEMPLATES
* Within TEMPLATES, select your template and click the "PROMPT ON LAUNCH" checkbox next to INVENTORY and EXTRA VARIABLES.

Template page

File: README.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+
+### JobTemplate
```
Comment (rlopez133) [inline README.md:158]: Curious do we not have this functionality for Workflows?

**Review by rlopez133 — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
+quay.io/ansible/awx-resource-operator# AWX Resource Operator
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://github.com/ansible/awx-resource-operator/workflows/CI/badge.svg?event=push)](https://github.com/ansible/awx-resource-operator/actions)
+[![Code of Conduct](https://img.shields.io/badge/code%20of%20conduct-Ansible-yellow.svg)](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) 
+[![AWX Mailing List](https://img.shields.io/badge/mailing%20list-AWX-orange.svg)](https://groups.google.com/g/awx-project)
+[![IRC Chat - #ansible-awx](https://img.shields.io/badge/IRC-%23ansible--awx-blueviolet.svg)](https://libera.chat)
+
+An kubernetes operator that can be used along side [Ansible AWX](https://github.com/ansible/awx) to launch automation jobs and create job templates by creating resources directly in kubernetes. This operator is built with [Operator SDK](https://github.com/operator-framework/operator-sdk) and Ansible.
```
Comment (rlopez133) [inline README.md]: A kubernetes operator....

File: README.md (hunk 7 above)
Comment (rooftopcellist) [reply README.md:158]: AnsibleJob's support launching job templates and workflow job templates, however I haven't tried using the JobTemplate CR to create WFJT's.  I expect more changes to the operator would be needed.

File: README.md (hunk 6 above)
Comment (rooftopcellist) [reply README.md]: ideally, we should add a screenshot of the new UI with PROMPT ON LAUNCH checked.

Comment (rooftopcellist) [reply README.md]: Ok, fixed now - https://github.com/ansible/awx-resource-operator/blob/61c5fbcc6f2e032a7867538e52e64352aaf85a40/docs/images/on-launch-configuration.png

File: README.md (hunk 8 above)
Comment (rooftopcellist) [reply README.md]: k8s ref removed.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+
+Note that prompt on launch needs to be enabled for inventories and extra vars if you are specifying those. In order to enable "PROMPT ON LAUNCH", within the AWX UI:
+
+1. Select Resources -> Templates
+2. Within Templates, select your template and click the "PROMPT ON LAUNCH" checkbox next to INVENTORY and EXTRA VARIABLES.
+
+![Prompt on Launch Configuration Example](./docs/images/on-launch-configuration.png "Prompt on Launch Configuration Example")
+
+
+#### Launching Workflows
```
Comment (rooftopcellist) [inline README.md:143]: @rlopez133 I also added a section here to show how to launch a workflow job.

**Pull request merged**